### PR TITLE
Jordan/3077 mobile form improvements (session and action modal)

### LINKED ISSUE #3077: improve session and action-modal forms on mobile
**Is your feature request related to a problem? Please describe.**

- the font sizes and buttons appear a bit small on mobile
- the button can sometimes be hidden and the scrolling is a bit awkward

**Describe the solution you'd like**

- make forms, buttons, text larger
- fixed back / cancel and button on bottom bar

### LINKED ISSUE #3101: Do not add "StatusBar" on desktop
**Describe the bug**

The `StatusBar` component is not used on desktop and creates frequent console errors. Use `config.mobileApp` to prevent this component from being added.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: 
 - Browser: 
 - Version: 

**Smartphone (please complete the following information):**
 - Device: 
 - OS: 
 - Browser 
 - Version 

**Additional context**

### PR DESCRIPTION
**Pull request by jbibla**
closes #3077
closes #3101

**Description:**
- fixed button to bottom on forms
- got rid of console error for status bar #3101
- fix validator warning color
- stop background scrolling in action modal
- added mobile config to session module
- cleaning up session and seed screens

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/components/common/TmSeed.vue [hunk 1]
```diff
@@ -1,5 +1,11 @@
 <template>
   <div class="seed-phrase">
+    <div class="seed-warning">
+      <p>
+        This backup code is all that is needed to access your account. Please
```
Comment (faboweb) [inline src/components/common/TmSeed.vue]: No mention of word order anymore?

File: src/components/staking/PageValidator.vue [hunk 2]
```diff
@@ -173,11 +173,11 @@ import { ValidatorProfile, UserTransactionAdded } from "src/gql"
 function getStatusText(statusDetailed) {
   switch (statusDetailed) {
     case "inactive":
-      return "Validator is currently not validating"
+      return "Validator is not currently validating"
     case "banned":
-      return "Validator is banned from the network"
+      return "Validator is temporarily banned from the network"
```
Comment (faboweb) [inline src/components/staking/PageValidator.vue]: nope, permanent

File: src/components/staking/PageValidator.vue [hunk 3]
```diff
@@ -173,11 +173,11 @@ import { ValidatorProfile, UserTransactionAdded } from "src/gql"
...
     default:
-      return "Validator is actively validating"
+      return "Validator is online and creating blocks"
```
Comment (faboweb) [inline src/components/staking/PageValidator.vue]: it is more about the rewards the validator receives I guess

**Comment by jbibla** [comment]
if we're going to use codecov again can we display the little graphic so it's easy to know what decreased?

File: src/ActionModal/components/ActionModal.vue [hunk 4]
```diff
@@ -415,6 +416,10 @@ export default {
     ...mapGetters([`connected`, `isExtensionAccount`]),
     ...mapGetters({ networkId: `network` }),
     checkFeatureAvailable() {
+      /* istanbul ignore next */
+      if (this.network.testnet) {
```
Comment (faboweb) [inline src/ActionModal/components/ActionModal.vue]: nop nop nop

File: src/ActionModal/components/ActionModal.vue [hunk 5]
```diff
@@ -438,7 +443,9 @@ export default {
     },
     signMethods() {
       let signMethods = []
-      if (this.session.sessionType === sessionType.EXPLORE) {
+      if (this.isMobileApp) {
```
Comment (faboweb) [inline src/ActionModal/components/ActionModal.vue]: is there a explore session on mobile? in that case, what would the sign page show?

File: src/components/common/TmFormStruct.vue [hunk 6]
```diff
@@ -54,8 +54,8 @@ export default {
 }
 
 .form-main {
-  display: flex;
-  flex-flow: column;
+  /* display: flex; */
```
Comment (faboweb) [inline src/components/common/TmFormStruct.vue]: remove comment?

File: src/ActionModal/components/ActionModal.vue (hunk 5 above)
Comment (jbibla) [reply src/ActionModal/components/ActionModal.vue]: yes there is. good question! i guess it should show a link to creating an account or signing in.

File: src/components/common/TmSeed.vue (hunk 1 above)
Comment (jbibla) [reply src/components/common/TmSeed.vue]: maybe this can be improved. but i think the numbers are quite clear. and most will copy to clipboard because of the button.

File: src/vuex/modules/session.js [hunk 7]
```diff
@@ -64,8 +64,8 @@ export default ({ apollo }) => {
     setExperimentalMode(state) {
       state.experimentalMode = true
     },
-    setInsecureMode(state) {
-      state.insecureMode = true
+    setInsecureMode(state, insecureMode) {
```
Comment (faboweb) [inline src/vuex/modules/session.js]: why this change? would there be any `commit("setInsecureMode", false)`?

File: src/styles/session.css [hunk 8]
```diff
@@ -167,25 +171,27 @@
...
+    margin: 1rem 1rem 3.5rem;
   }
 
   .session-footer {
+    position: fixed;
+    bottom: 0;
+    left: 0; 
+    right: 0;
+    width: 100%;
+    /* flex-direction: column; */
```
Comment (jbibla) [inline src/styles/session.css]: delete
